.github/workflows/maven.yml

```diff
@@ -32,4 +32,4 @@ jobs:
         distribution: 'temurin'
         cache: maven
     - name: Build with Maven
-      run: mvn -B package -Dspring.profiles.active=dev --file pom.xml
+      run: mvn -B package -Dspring.profiles.active=test --file pom.xml
```

Comment:
i created a dedicated for tests test profile, and deleted the dev profile as it was no longer needed